en/messages.json

```diff
@@ -446,6 +453,9 @@
     "close": {
         "message": "Close"
     },
+    "returnToPreviousPanel": {
+        "message": "Return to previous panel"
+    },
```

Comment:
Alt tag for this back button/arrow: 

<img width="78" alt="image" src="https://user-images.githubusercontent.com/2692333/224164547-891e9927-deb6-4d61-97b9-dc19839e25a8.png">